packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.story.internal.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 
-import TableInputCSV from './TableInputCSV';
+import { TableInputCSV } from './TableInputCSV';
+import { Meta } from '@storybook/react/types-6-0';
```

Comment:
Minor - this is likely an auto import but there shouldn't be a need to reference `/types-6-0` here. `import { Meta } from '@storybook/react` should be enough.

Reply:
Good catch Jack!